addons/web/tooling/_eslintrc.json

```diff
@@ -43,24 +43,27 @@
     "_": "readonly",
     "Chart": "readonly",
     "fuzzy": "readonly",
-    "Popover": "readonly",
     "StackTrace": "readonly",
     "QUnit": "readonly",
     "luxon": "readonly",
     "py": "readonly",
     "FullCalendar": "readonly",
     "globalThis": "readonly",
-    "Modal": "readonly",
-    "Dropdown": "readonly",
     "ScrollSpy": "readonly",
-    "Tooltip": "readonly",
-    "Collapse": "readonly",
-    "Alert": "readonly",
     "module": "readonly",
     "chai": "readonly",
     "describe": "readonly",
     "it": "readonly",
+    "afterEach": "readonly",
```

Comment:
This one was in the other file but not here

Reply:
This one should probably not be treated as a global everywhere because it's only available in the OdooEditor's test suite, it should have been added as a comment at the top of that file instead of being added as a global everywhere. This should probably only contain libraries that are loaded basically everywhere.

Reply:
Thanks for your remarks.
I actually rewrote the diff history, instead of one commit there is multiple ones. It will be easier to figure.
Also applied your suggestions. Let me know if it doesn't seem right.